File: branches/Medusa/Source/Fantasy.Jobs/Try.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Fantasy.XSerialization;

namespace Fantasy.Jobs
{
    [Instruction]
    [XSerializable("try", NamespaceUri = Consts.XNamespaceURI)]  
    internal class Try : Sequence, IConditionalObject 
    {
        public override void Execute()
        {
            base.ExecuteSequence();
        }


        [XAttribute("condition")]
        public string Condition { get; set; }
       
    }
}

File: Fantasy.Jobs/src/fantasy/jobs/Try.java
package fantasy.jobs;

import fantasy.xserialization.*;

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
@Instruction
@XSerializable(name = "try", namespaceUri = Consts.XNamespaceURI)
public class Try extends Sequence implements IConditionalObject
{
	@Override
	public void Execute() throws Exception
	{
		this.ExecuteSequence();
	}

	@XAttribute(name = "condition")
	private String _condition = null;
	public String getCondition()
	{
		return this._condition;
	}
	public void setCondition(String value)
	{
		this._condition = value;
	}

}